ui/src/timeMachine/utils/queryBuilder.ts

```diff
@@ -47,14 +47,20 @@ export const isDraftQueryAlertable = (
 }
 
 export const isCheckSaveable = (
```

Comment:
hrmm. I wonder if there are other things we should check for here? Maybe for another PR.